tests/rpc/test_rpc_apiserver.py

```diff
@@ -463,13 +463,13 @@ def test_api_count(botclient, mocker, ticker, fee, markets):
     assert rc.json()["current"] == 4
     assert rc.json()["max"] == 1
 
-    ftbot.config['max_open_trades'] = float('inf')
+    freqtrade.config['max_open_trades'] = float('inf')
     rc = client_get(client, f"{BASE_URI}/count")
     assert rc.json()["max"] == -1
 
 
 def test_api_locks(botclient):
-    ftbot, client = botclient
+    freqtrade, client = botclient
```

Comment:
While i apreciate you taking the time to align all our namings - for next time (should you notice something similar) please do this as a seperate PR (simply extract this part as a single commit on top of develop and do a "quick" PR).

That way, it'll become clear which changes are actually caused by the change / feature / ... you're suggesting / adding - and which are simple refactorings / renamings (which will in term make reviewing it easier).

Now i've looked through it though - so for this PR, leave it in please :+1: